### PR TITLE
Integration tests

### LINKED ISSUE #50: Add integration tests with playwright
While testing and reviewing PRs I typically go through a couple of steps:

* Generate an experiment with default settings
* Add a permutation that changes the initial height to 800
* Create a timeseries plot
* Duplicate the experiment and change the entrainment ratio to 0.3 in the reference
* Verify that we have 4 lines, two starting from 200m, two starting from 800m, like so:

image

It would be nice to record this workflow and add automated tests to assure that we don't accidentily break things. We could think about other typical or edge-case workflows that should be tested as well.

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #50 

TODO
* [x] automate scenario steps of issue in a test
* [x]  fix code to pass test
* [x] test other permutation actions
* [x] add aria attributes to make playwright tests more readable (and improve accessibility)

To test
```
cd app/class-solid
pnpm exec playwright test
pnpm exec playwright show-report
```
Expand `Duplicate experiment with a permutation` test and scroll to bottom to see the plot.

d56cc5cc691adb07f801de3558bdad440fa833bd

### CODE REVIEW

**Comment by sverhoeven** [comment]
The test goes wrong because there are full configs everywhere and changing a reference keeps its permutations unchanged. So permutation in both experiments are the same.

File: README.md [hunk 1]
```diff
@@ -95,6 +95,14 @@ pnpm exec playwright install  # first time only
 pnpm exec playwright test
 ```
 
+To develop and debug end-to-end tests use
+
+```shell
+pnpm exec playwright test --ui --headed
```
Comment (Peter9192) [inline README.md:101]: Perhaps add as "test" to package.json?

**Review by Peter9192 — APPROVED** [review]
Supertoll!
Would be nice to have a bit clearer/more condensed overview of what each test actually requires, now it's a bit spaced out.

File: README.md (hunk 1 above)
Comment (sverhoeven) [reply README.md]: done in 6f5dbbef843e5fa1531e26a661c08c688776e443

**Comment by sverhoeven** [comment]
> Supertoll! Would be nice to have a bit clearer/more condensed overview of what each test actually requires, now it's a bit spaced out.

The comment above each block should make it clearer
